src/WebSdk/Web/Sdk/Sdk.props

```diff
@@ -73,11 +73,18 @@ Copyright (c) .NET Foundation. All rights reserved.
     <Using Include="Microsoft.Extensions.Logging" />
   </ItemGroup>
 
+  <!--Built in Microsoft.NET.Build.Containers version is same as SDK version-->
   <PropertyGroup>
-    <SdkContainerSupportPackageVersion>0.4.0</SdkContainerSupportPackageVersion>
+    <SdkContainerSupportPackageVersion Condition="'$(SdkContainerSupportPackageVersion)' == ''">$(NETCoreSdkVersion)</SdkContainerSupportPackageVersion>
```

Comment:
This is a good change. it will ensure that we can be on the latest and greatest. Thanks for making this happen.